spec/feature/authenticated_user_spec.rb

```diff
@@ -63,21 +71,15 @@
   end
 
   it "can view their own page" do
-    user = User.create(first_name: "Alice",
-                       last_name: "Smith",
-                       email: "rich.shea@gmail.com",
-                       password: "password")
-    allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(user)
-    visit user_path(user)
-    expect(current_path).to eq(user_path(user))
+    allow_any_instance_of(ApplicationController).to receive(:current_user).
+                                                    and_return(valid_user)
+    visit user_path(valid_user)
+    expect(current_path).to eq(user_path(valid_user))
```

Comment:
user moved to a let block with new variable `valid_user` reducing duplication and making the `valid_user_logs_in` helper method possible.